particles/Demo/Demo.recipes

```diff
@@ -1,5 +1,6 @@
 import 'DemoData.manifest'
+import 'TVMazeDemo.recipes'
+
 //import 'RestaurantsDemo.recipes'
 //import 'ProductsDemo.recipe'
-import 'TVMazeDemo.recipes'
```

Comment:
Does this change make a difference? I haven't touched the comment parser yet but can take a look (as to me this looks like it should work).

Reply:
There is no semantic change here.